lighthouse-core/scripts/chromium-web-tests/download-blink-tools.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+##
+# @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+##
+
+# Download blink tools for run_web_tests.py.
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+commit_chromium="41f18f569f022d78337f586dacc6eff0e0477e46"
+commit_catapult="95c1f426155576790a73778571c34a0f3cd6d608"
+
+if [ x"$BLINK_TOOLS_PATH" == x ]; then
+  echo "Error: Environment variable BLINK_TOOLS_PATH not set"
+  exit 1
+fi
+
+if [ -e "$BLINK_TOOLS_PATH" ]; then
```

Comment:
when commit hashes change above will local runs need to be cleared out? probably worth mentioning in the readme

Reply:
Could make the commit hash part of the installation path, and symlink a `cur` folder for usage in the `run_web_tests.sh`